roles/openshift_health_checker/test/docker_storage_test.py

```diff
@@ -56,14 +56,35 @@ def test_is_active(is_containerized, group_names, is_active):
     (
         dict(info={
             "Driver": "overlay2",
-            "DriverStatus": []
+            "DriverStatus": [("Backing Filesystem", "xfs")],
         }),
         False,
         [],
     ),
     (
         dict(info={
             "Driver": "overlay",
+            "DriverStatus": [("Backing Filesystem", "btrfs")],
+        }),
+        True,
+        ["'xfs' as the backing storage"],
```

Comment:
Reading the test, how is one to tell what the file systems are?! I saw `"btrfs"` in one place, and was confused seeing `"'xfs' ...` nearby.